programmers/20210503/문자열-내-p와-y의-개수-세기.js

```diff
@@ -0,0 +1,13 @@
+function solution(s){
+    var count = 0;
```

Comment:
변수 선언 var, let, const의 차이에 대해서 공부해보셔야 합니다.
```suggestion
    let count = 0;
```